### PR TITLE
add facebook sharer

### LINKED ISSUE #447: facebook share
We want this.

### PR DESCRIPTION
**Pull request by myarjunar**
fix #447 
add facebook sharer
post-to-fb-feti

### CODE REVIEW

File: django_project/feti/static/js/scripts/share.js [hunk 1]
```diff
@@ -68,6 +68,19 @@ define(['backbone', 'jquery', 'underscore'], function (Backbone, $, _) {
...
             });
         };
+        
+        this.shareToFacebook = function () {
+            // get url
+            var host = Backbone.history.location.host;
+            var windowReference = window.open('', '_blank', 'location=yes,height=570,width=520,scrollbars=yes,status=yes');
+
+            _generateURL(function (data) {
+                var facebook_sharer = 'http://www.facebook.com/sharer/sharer.php?s=100&p' +
```
Comment (dimasciput) [inline django_project/feti/static/js/scripts/share.js]: IIRC, this won't work if we have ssl installed.

Comment (dimasciput) [reply django_project/feti/static/js/scripts/share.js]: @myarjunar

Comment (myarjunar) [reply django_project/feti/static/js/scripts/share.js]: will changing it to `https` do the trick?

Comment (dimasciput) [reply django_project/feti/static/js/scripts/share.js]: @myarjunar try write the snippet here https://jsfiddle.net, then test it

**Review by dimasciput — APPROVED** [review]

**Pull request merged**